### PR TITLE
strcmp to == conversions

### PR DESCRIPTION
**Pull request by neheb**
Does the same thing.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Review by piponazo — APPROVED** [review]
LGTM

File: src/tiffcomposite_int.cpp [hunk 1]
```diff
@@ -24,8 +24,8 @@ uint32_t fillGap(Exiv2::Internal::IoWrapper& ioWrapper, uint32_t curr, uint32_t
 // class member definitions
 namespace Exiv2::Internal {
 bool TiffMappingInfo::operator==(const TiffMappingInfo::Key& key) const {
-  return (0 == strcmp("*", make_) || 0 == strncmp(make_, key.m_.c_str(), strlen(make_))) &&
```
[the remark this answers is not in the document]
Comment (kevinbackhouse) [reply src/tiffcomposite_int.cpp:27]: @neheb: I accidentally clicked "start a review" before, so this comment was left in a pending state.

[the remark this answers is not in the document]
Comment (kevinbackhouse) [reply src/tiffcomposite_int.cpp:27]: I've created #2307 to use `startsWith` here.

[the remark this answers is not in the document]
Comment (neheb) [reply src/tiffcomposite_int.cpp:27]: Ah strncmp, yes.

**Pull request merged**